### ISSUE TITLE
Please depend on 'igraph' instead of 'python-igraph'

### ISSUE THREAD

**Issue by szhorvat** [scikit-mobility/scikit-mobility#268]
Please depend on `igraph` instead of `python-igraph`. The latter name is deprecated on PyPI. I believe changes are needed in these two locations:

https://github.com/search?q=repo%3Ascikit-mobility%2Fscikit-mobility%20python-igraph&type=code

See https://github.com/igraph/python-igraph/issues/699 for an explanation.

**Comment by me9hanics** [comment]
I also just had a dependency issues solely because `python-igraph` required a different version of a package than another package I have required. 

An extra note is that they should make the dependencies less rigid, while installing the library in one of my environments I just noticed how many downgrades it makes, basically downgrades all used libraries to a quite old version - of course most do not even need downgrading. The amount of packages `skmob` depends on is way too much, while installing I have not seen so many required packages for just installing one. I think they should try to integrate some less core package imports into their own code too.

**Comment by szhorvat** [comment]
Yes, the very old igraph version that is pinned now (0.9.1, released over 3 years ago) does not have binary wheels for recent Python versions, so people who try to install it will need to compile from sources. This won't work out of the box for the majority of Windows users.
